content/_index.md

```diff
@@ -0,0 +1,64 @@
++++
+title = "Docsy"
+linkTitle = "Docsy"
+
++++
```

Comment:
Why the "Docsy" `title` and `linkTitle`?
(If these fields are actually required, can we stick to YAML frontmatter?)

Reply:
Fixed! Leaving this page's frontmatter empty.